src/Components/ArtistFilter/Filter.tsx

```diff
@@ -7,7 +7,7 @@ import ArtworksContent from "./Artworks"
 
 import { Filter_artist } from "../../__generated__/Filter_artist.graphql"
 
-interface Props {
+export interface Props {
```

Comment:
This was b/c of a TS lint error, we are exporting a class which wraps this [private] interface.